----------------------------------------------------------------------
Timestamp - 03/03/2022 12:51:29 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 15700
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 12:51:59 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 15700
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 12:52:29 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 15700
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 12:53:31 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "8000000e-0002-fa00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|44986802-41f4fbe5d91ef9a5."
TraceId : "44986802-41f4fbe5d91ef9a5"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 15700
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 12:57:26 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 3428
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 12:57:50 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 3428
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 12:58:17 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 3428
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 12:59:16 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 20948
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 12:59:34 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 20948
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:00:03 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 20948
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:25:55 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "Could not find a part of the path 'C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\wwwroot\uploads\user\profile\default\profile.jpeg'."
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12132
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:25:58 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "Could not find a part of the path 'C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\wwwroot\uploads\user\profile\default\profile.jpeg'."
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12132
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:26:21 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12132
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:26:45 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12132
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:27:59 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "8000000e-0001-fa00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|95002359-441ad75bdf68a872."
TraceId : "95002359-441ad75bdf68a872"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12132
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:29:02 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 9684
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:29:27 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 9684
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:30:41 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "8000000f-0000-f900-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|362065db-4c142f5eb845f582."
TraceId : "362065db-4c142f5eb845f582"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 9684
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:40:34 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12392
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 03/03/2022 13:40:56 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12392
ProcessName : "iisexpress"
---------------------------------------------------------------------------
